collect_app/src/main/res/drawable/ic_visibility_off.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="36dp" android:tint="#9F9F9F"
```

Comment:
Why is the tint required? We can directly put this color hex to `fillColor` as well, right?

Reply:
I downloaded that icon using Android Studio so I didn't change anything. I think it's ok as lng as we use the same color for both themes. If we want to use dfferent colors in the furure we can change it.